tests/__tests__/api/user/getEmail.js

```diff
@@ -23,27 +23,25 @@ beforeEach(async () => {
   execSync("npm run reset-s", { stdio: "ignore" });
 });
 
-describe("Get User By Username Tests", () => {
-  test("get a user by a valid/existing username raisa", async () => {
-    const response = await request(app).get("/user/username/raisa");
+describe("Get User By Email Tests", () => {
+  test("get a user by a valid/existing email (Raisa)", async () => {
+    const response = await request(app).get("/user/email/raisa@raisa.com");
```

Comment:
URLs with special characters must be encoded properly. Please refer to this chart https://www.w3schools.com/tags/ref_urlencode.asp
```suggestion
    const response = await request(app).get("/user/email/raisa%40raisa.com");
```